package-scripts.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+const path = require('path');
+
+const MOCHA = `node ${path.join('bin', 'mocha')}`;
+const TM_BUNDLE = 'JavaScript\\ mocha.tmbundle';
+
+function test (testName, params) {
+  const prefix = process.env.COVERAGE ? `nyc --no-clean --report-dir coverage/reports/${testName}` : '';
+  return `${prefix} ${MOCHA} ${params}`;
+}
+
+module.exports = {
```

Comment:
I would absolutely love it if we added description fields to everything here

Reply:
Kk — check over them in case anything is off. 👍